websocket/src/websocket.cpp

```diff
@@ -449,6 +462,68 @@ HandshakeResponse::~HandshakeResponse()
 }
 
 
+
+// ***************************************************************************************************
+// Emscripten Websocket library callbacks
```

Comment:
We have `wslay_callbacks.cpp` and I suggest you add "emscripten_callbacks.cpp"

Reply:
Done!